gadget/install/install.go

```diff
@@ -176,12 +176,18 @@ func Run(gadgetRoot, device string, options Options) error {
 		loadChain = append(loadChain, options.KernelPath)
 	}
 
-	// TODO:UC20: get cmdline definition from bootloaders
+	// Get kernel command lines
+	cmdline, err := boot.ComposeCommandLine(options.Model)
+	if err != nil {
+		return fmt.Errorf("cannot obtain kernel command line: %v", err)
+	}
+	recoveryCmdline, err := boot.ComposeRecoveryCommandLine(options.Model, options.SystemLabel)
```

Comment:
Warrants a comment that this is an expected kernel command line of a recovery system that we are installing from.